domain/camera/src/main/java/com/google/jetpackcamera/domain/camera/CameraXCameraUseCase.kt

```diff
@@ -189,6 +192,29 @@ class CameraXCameraUseCase @Inject constructor(
         )
     }
 
+    override fun tapToFocus(
+        display: Display,
+        surfaceWidth: Int,
+        surfaceHeight: Int,
+        x: Float,
+        y: Float
+    ) {
+        if (camera != null) {
+            val meteringPoint = DisplayOrientedMeteringPointFactory(
+                display,
+                camera!!.cameraInfo,
```

Comment:
avoid `!!` here, you can just do `camera.cameraInfo` since camera is not null here

Reply:
could not do that, the error message read: `Smart cast to 'Camera' is impossible, because 'camera' is a mutable property that could have been changed by this time`